src/components/ConditionalFilter/filterDefinitions/definitions/AttributeDefinition.ts

```diff
@@ -0,0 +1,160 @@
+import { ApolloClient } from "@apollo/client";
+import { AttributeInput, AttributeInputTypeEnum } from "@dashboard/graphql";
+
+import { AttributeChoicesHandler, Handler } from "../../API/Handler";
+import { FilterElement } from "../../FilterElement";
+import { WhereOnlyFilterDefinition } from "../types";
+import { extractConditionValueFromFilterElement, getBooleanValueFromElement } from "../utils";
+
+export class AttributeDefinition
+  implements WhereOnlyFilterDefinition<{ attributes?: AttributeInput[] }>
+{
+  public canHandle(element: FilterElement): boolean {
+    return element.rowType() === "attribute";
+  }
+
+  public createOptionFetcher(
+    client: ApolloClient<unknown>,
+    inputValue: string,
+    element: FilterElement,
+  ): Handler {
+    const { value: id } = element.selectedAttribute || element.value;
+
+    return new AttributeChoicesHandler(client, id, inputValue);
+  }
+
+  public updateWhereQuery(
+    query: Readonly<{ attributes?: AttributeInput[] }>,
```

Comment:
It's variables that are being updated, not query, so maybe lets be explicit on names